server/meshmodel/istio-base/1.16.0/v1.0.0/relationships/hierarchical-inventory-WASMFilter-to-envoyPlugin.json

```diff
@@ -3,7 +3,8 @@
   "version": "v1.0.0",
   "kind": "Hierarchical",
   "metadata": {
-    "description": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component."
+    "description": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component.",
+    "type": "SingularNodeInventoryWallet"
```

Comment:
Eventually, this value will move to `subType`, right?

`type` will become "child" at that point?

Reply:
yup , this will be the subtype